src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int weight;
```

Comment:
The field 'weight' should be accessed through getters and setters. It is a good practice to keep fields private and provide public methods to access them. This encapsulates the internal representation and allows for validation and control over how the fields are accessed or modified.